spark-2.0-on-suse/azuredeploy.json

```diff
@@ -23,6 +23,57 @@
 	  "minValue": 1,
       "maxValue": 50
     },
+    "sparkSlavesSize": {
```

Comment:
@bmoore-msft: I agree with you, there is also another use case fro training for example, where people would spin-up the template without investigating what could be a specific machine features. Could be t-shirt size another good option?